Makefile

```diff
@@ -342,16 +347,27 @@ switch-eks:
 #################################
 ##  --    Docker images    --  ##
 #################################
+docker-multiarch-build: go-generate generate-config-file 
+	@ hack/docker.sh -l -m $(OPERATOR_IMAGE)
+	docker buildx build . \
+		--progress=plain \
+		--build-arg GO_LDFLAGS='$(GO_LDFLAGS)' \
+		--build-arg GO_TAGS='$(GO_TAGS)' \
+		--build-arg VERSION='$(VERSION)' \
+		--platform linux/amd64,linux/arm64 \
+		--push \
+		-t $(OPERATOR_IMAGE)
 
-docker-build: go-generate generate-config-file
-	docker build . \
+docker-build: go-generate generate-config-file 
+	DOCKER_BUILDKIT=1 docker build . \
```

Comment:
I had to introduce it because the normal `docker build` command could not parse the changes made to the Dockerfile (specifically, the `--platform` argument). The alternative would have been to use two Dockerfiles -- which didn't seem right.